griptape/drivers/embedding/base_embedding_driver.py

```diff
@@ -12,23 +12,25 @@
 class BaseEmbeddingDriver(ExponentialBackoffMixin, ABC):
     """
     Attributes:
+        model: The name of the model to use.
         tokenizer: An instance of `BaseTokenizer` to use when calculating tokens.
-        chunker: An instance of `BaseChunker` to use when chunking long strings.
     """
 
-    tokenizer: BaseTokenizer = field(kw_only=True)
+    model: str = field(kw_only=True)
+    tokenizer: BaseTokenizer | None = field(kw_only=True)
```

Comment:
1. `Optional[BaseTokenizer]`
2. Should it have `None` as the default?

Reply:
1. This change comes from pyupgrade: https://github.com/asottile/pyupgrade?tab=readme-ov-file#pep-604-typing-rewrites
2. Yep.